ship: refactor teams to match reorg

nelsonjr opened the pull request:
Reorganize handbook documentation to match reorg of June, 2023.

Commit:
Refactor teams to match reorg.

Commit:
Refactor teams to match reorg.

Commit:
Refactor teams to match reorg.

nelsonjr committed:
squash! Prettier

Commit:
Update roles instead.

Commit:
fix links

Commit:
fix team links

michaellzc commented on content/departments/engineering/dev/tools/scaletesting.md:
```diff
@@ -52,10 +52,10 @@ The configuration for the Google Cloud infrastructure can be found in the [sourc
 
 To access this instance, run the following command:
-`gcloud compute ssh --zone "us-central1-a" "devx" --tunnel-through-iap --project "sourcegraph-scaletesting"`
+`gcloud compute ssh --zone "us-central1-a" "DevInfra" --tunnel-through-iap --project "sourcegraph-scaletesting"`
```
this is reference to a GCE instance, we shouldn't rename it.

michaellzc commented on content/departments/cloud/technical-docs/v2.0/index.md:
```diff
@@ -72,7 +72,7 @@ All customer instances are considered part of the `prod` environment and all cha
 ##### [s2 instance](https://sourcegraph.sourcegraph.com/)
 
-This is the internal Cloud dogfood instance for the entire company. #dev-experience is responsible for rolling out nightly builds on this instance and #cloud is responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
+This is the internal Cloud dogfood instance for the entire company. #devinfra is responsible for rolling out nightly builds on this instance and #cloud is responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
```
Historically DevX is the official owner of CD infra for S2. This is less true now because

- Cloud owns the upcoming Cloud release channel Job Fair project
- In the previous quarter, Cloud has been adding new features to the CD infra, e.g, continues upgrade that tracks specific release branch on another Cloud instance, cody-dev.sgdev.dev.

```suggestion
This is the internal Cloud dogfood instance for the entire company. #ask-cloud-ops is responsible for rolling out nightly builds on this instance. Additionally, they are responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
```

michaellzc commented on content/departments/cloud/index.md:
```diff
@@ -228,13 +228,13 @@ GCP Logging is used to store / query logs written by Sourcegraph workloads, and
 Cloud instances provisioned for customers run [released](../engineering/dev/process/releases/index.md#releases) Sourcegraph versions and are currently updated at least once a month (for minor releases), unless we need to deploy a patch release.
 
-Sourcegraph-owned instances are continuously deployed (with versions that weren't officially released), [DevX team](../engineering/teams/dev-experience/index.md) owns continuous deployment to those environments.
+Sourcegraph-owned instances are continuously deployed (with versions that weren't officially released), [Developer Infrastructure team](../engineering/teams/devinfra/index.md) owns continuous deployment to those environments.
```
```suggestion
Sourcegraph-owned instances can opt-in to receive continuous upgrades with versions that weren't officially released, e.g., the latest commit on `main` branch, release branches. We own continuous deployment to those environments.
```

michaellzc commented on content/departments/cloud/index.md:
```diff
@@ -1,12 +1,12 @@
 
 > NOTE: **Cloud means single-tenant dedicated instances managed by Sourcegraph** _(for example `mycompany.sourcegraph.com`)_. Sourcegraph Cloud should not be confused with Sourcegraph.com which holds public and open source code. The Cloud and managed instance should be considered synonyms within these handbook pages.
 
 The Cloud team is the special focus team reporting directly to CEO modeled on _“if AWS were to offer ‘Managed Sourcegraph’ like they do Elasticsearch, Redis, PostgreSQL, etc., how would they do it?”_ The team is responsible for maintaining existing [managed instances](https://docs.sourcegraph.com/admin/install/managed) and building the next generation of them. The Cloud team has no other responsi ...
```
some feedback to the Cloud section:

I think in the long run, we want Cloud to become the platform team and offload On-Call responsibility back to the home team. Is this something we want to capture here?

nelsonjr replied:
We can. This is the team area, so as long as the team and leadership are aligned, it sounds like a good idea.

rafax commented on content/departments/cloud/index.md:
```diff
@@ -1,12 +1,12 @@
-# Cloud
+# Cloud Opreations Team
```
```suggestion
# Cloud Operations Team
```

sanderginn replied:
```suggestion
# Cloud Operations Team
```

sanderginn commented on content/departments/engineering/dev/index.md:
```diff
@@ -91,7 +91,7 @@ The current channels are:
 - #dev-chat
 - #dev-ops
-- #dev-experience
+- #devinfra
```
This is not correct, right? The team is DevInfra, the channel is `#team-dev-infra`, and `#devinfra` doesn't exist

nelsonjr replied:
As many channels are being revisited, we must do a pass. I will hope the teams will update their pages in a crowd-sourced way.

unknwon replied:
Wrong info is worse than no info, i'm updating/deleting this list to reflect the current truth for now

sanderginn commented on content/departments/engineering/dev/process/deployments/instances.md:
```diff
@@ -7,7 +7,7 @@ Information about Sourcegraph's different instances.
   - [demo.sourcegraph.com](instances.md#demosourcegraphcom) is a managed instance used for CE demos.
   - [devmanaged.sourcegraph.com](instances.md#devmanagedsourcegraphcom) is a managed instance used for managed instances development.
-  - [sourcegraph.sourcegraph.com](instances.md#sourcegraphsourcegraphcom-s2) is a managed instance used for Sourcegraph's dogfooding needs. Informally referred to as "S2". This instance is jointly owned by #team-dev-experience and #team-cloud. We [continously deploy](https://golinks.io/s2-deploy) the latest unreleased version of Sourcegraph every one hour on weekdays, and you may also [force roll ...
+  - [sourcegraph.sourcegraph.com](instances.md#sourcegraphsourcegraphcom-s2) is a managed instance used for Sourcegraph's dogfooding needs. Informally referred to as "S2". This instance is jointly owned by #team-devinfra and #team-cloud. We [continously deploy](https://golinks.io/s2-deploy) the latest unreleased version of Sourcegraph every one hour on weekdays, and you may also [force rolling ou ...
```
```suggestion
  - [sourcegraph.sourcegraph.com](instances.md#sourcegraphsourcegraphcom-s2) is a managed instance used for Sourcegraph's dogfooding needs. Informally referred to as "S2". This instance is jointly owned by #team-dev-infra and #team-cloud. We [continously deploy](https://golinks.io/s2-deploy) the latest unreleased version of Sourcegraph every one hour on weekdays, and you may also [force rolling out a new deployment](https://golinks.io/s2-deploy) by clicking the `run workflow` button and using `sg` as the customer slug.
```

sanderginn commented on content/departments/engineering/dev/process/engineering_ownership.md:
```diff
@@ -129,7 +129,7 @@ If you see an area that is missing, [figure out](../../product/process/feedback/
 - sourcegraph/docsite
 
-## [Delivery](../../teams/delivery/index.md) (and Delivery Tiger Team)
+## [Delivery](../../teams/release/index.md) (and Delivery Tiger Team)
```
Should the header change to `Release`?

jdpleiness replied:
```suggestion
## [Delivery](../../teams/release/index.md)
```
Removing references to the Delivery Tiger Team

sanderginn commented on content/departments/engineering/dev/process/engineering_ownership.md:
```diff
@@ -156,7 +156,7 @@ If you see an area that is missing, [figure out](../../product/process/feedback/
 - [sourcegraph/worker](https://github.com/sourcegraph/sourcegraph/tree/main/cmd/worker)
 
-## [Developer Experience](../../teams/dev-experience/index.md)
+## [Developer Experience](../../teams/devinfra/index.md)
```
Should the header change to `DevInfra`?

unknwon replied:
```suggestion
## [Developer Infrastructure](../../teams/devinfra/index.md)
```

sanderginn commented on content/departments/engineering/dev/process/engineering_ownership.md:
```diff
@@ -397,4 +397,4 @@ OWNERS files and CODENOTIFY files use the same syntax, and notifications for bot
 - If there is an individual with expertise in a given part of the code, who is not on the owning team (e.g., Thorsten Ball with sg), that person should be listed in OWNERS.
 
-For questions about our use of owners files, please reach out on #dev-experience.
+For questions about our use of owners files, please reach out on #devinfra.
```
```suggestion
For questions about our use of owners files, please reach out on #discuss-dev-experience.
```

Note that `#ask-dev-infra` actually doesn't exist, it's still called `#ask-dev-experience`

sanderginn commented on content/departments/engineering/dev/process/incidents/playbooks/ci.md:
```diff
@@ -39,7 +39,7 @@ In order to handle problems with the CI, the following elements are necessary:
 #### Actions
 
-`buildchecker` will still allow the authors of the last few failed builds, as well as the @dev-experience team, to push to the `main` branch so as to make any changes necessary to restore the pipeline to a healthy state.
+`buildchecker` will still allow the authors of the last few failed builds, as well as the @devinfra team, to push to the `main` branch so as to make any changes necessary to restore the pipeline to a healthy state.
```
The `@devinfra` team still needs to be created/renamed from devex

unknwon replied:
```suggestion
`buildchecker` will still allow the authors of the last few failed builds, as well as the @dev-infra team, to push to the `main` branch so as to make any changes necessary to restore the pipeline to a healthy state.
```

sanderginn commented on content/departments/engineering/dev/process/incidents/playbooks/ci.md:
```diff
@@ -137,7 +137,7 @@ In order to handle problems with the CI, the following elements are necessary:
 
 1. Escalate by creating an incident (`/incident` on Slack).
-1. Get some help by pinging `@dev-experience-support` on Slack in the #buildkite-main or #dev-experience channels.
+1. Get some help by pinging `@devinfra-support` on Slack in the #buildkite-main or #devinfra channels.
```
```suggestion
1. Get some help by pinging `@dev-infra-support` on Slack in the #buildkite-main or #discuss-dev-infra channels.
```

Same as previous comment on channels/handles.

sanderginn commented on content/departments/engineering/dev/process/incidents/playbooks/ci.md:
```diff
@@ -158,7 +158,7 @@ In order to handle problems with the CI, the following elements are necessary:
   - 💡 This should really not be happening.
   - Is the error about the Docker daemon?
-    - **Yes**, this is a CI infrastructure flake. Ping `@dev-experience-support` on Slack in the #buildkite-main or #dev-experience channels.
+    - **Yes**, this is a CI infrastructure flake. Ping `@devinfra-support` on Slack in the #buildkite-main or #devinfra channels.
```
```suggestion
    - **Yes**, this is a CI infrastructure flake. Ping `@dev-infra-support` on Slack in the #buildkite-main or #discuss-dev-infra channels.
```

sanderginn commented on content/departments/engineering/dev/process/incidents/playbooks/ci.md:
```diff
@@ -169,7 +169,7 @@ In order to handle problems with the CI, the following elements are necessary:
   - Missing tools that we use to run the steps, such as `go`, `node`, `comby`, ...
   - Errors from `asdf`, which is used to manage the above tools.
-- **Yes**: ping `@dev-experience-support` on Slack in the #buildkite-main or #dev-experience channels.
+- **Yes**: ping `@devinfra-support` on Slack in the #buildkite-main or #devinfra channels.
```
```suggestion
- **Yes**: ping `@dev-infra-support` on Slack in the #buildkite-main or #discuss-dev-infra channels.
```

sanderginn commented on content/departments/engineering/dev/process/incidents/playbooks/ci.md:
```diff
@@ -218,7 +218,7 @@ You can also refer to the [Loom walkthrough "how to find out if a CI failure is
 
 1. Inspect [webhooks status](https://github.com/sourcegraph/sourcegraph/settings/hooks) on the `sourcegraph/sourcegraph` repository settings
-1. If you're not authorized to see this page, ping `@dev-experience-support` or escalate to `@github-owners`.
+1. If you're not authorized to see this page, ping `@devinfra-support` or escalate to `@github-owners`.
```
```suggestion
1. If you're not authorized to see this page, ping `@dev-infra-support` or escalate to `@github-owners`.
```

rafax committed:
Update Cloud main page

sanderginn commented on content/departments/engineering/dev/process/pullrequest-compliance.md:
```diff
@@ -2,26 +2,26 @@
 All pull requests must provide [a test plan](https://docs.sourcegraph.com/dev/background-information/testing_principles#test-plans) and be reviewed by a teammate. If for situational reasons, it needs to be exempted, an [exception is automatically recorded](https://docs.sourcegraph.com/dev/background-information/testing_principles#exceptions).
 
-Exceptions are periodically reviewed by the @dev-experience-team on a weekly basis.
+Exceptions are periodically reviewed by the @devinfra-team on a weekly basis.
```
```suggestion
Exceptions are periodically reviewed by the @dev-infra-team infra-team on a weekly basis.
```

Is this actually true?

sanderginn commented on content/departments/engineering/dev/process/pullrequest-compliance.md:
```diff
@@ -2,26 +2,26 @@
 > All issues prior to 2023-05-22 have been reviewed manually, even if they do not carry the `approved` label.
 
-1. Current engineer on @dev-experience-support rotation reviews closed issues which do not have the label `approved`.
+1. Current engineer on @devinfra-support rotation reviews closed issues which do not have the label `approved`.
```
Same q, is this actually done?

sanderginn commented on content/departments/engineering/dev/tools/infrastructure/gcp.md:
```diff
@@ -58,7 +58,7 @@ Compute Engine lets you purchase committed use contracts in return for deeply di
 ### Current commitments
 
-We do not have any ongoing commitments as of right now since the most recent commitment expired. The [DevExp](../../../teams/dev-experience/index.md) team is currently investigating resource usage as part of an effort to rightsize GCP consumption.
+We do not have any ongoing commitments as of right now since the most recent commitment expired. The [DevExp](../../../teams/devinfra/index.md) team is currently investigating resource usage as part of an effort to rightsize GCP consumption.
```
Not sure if this is still accurate. I think we have committed discounts atm (@rafax ?)

unknwon replied:
This entire page contains full of outdated content 😂 it needs ultra-revamp. I'll make a note and come back to this page later.

sanderginn commented on content/departments/engineering/teams/devinfra/newsletter.md:
I don't know if we should rename occurrences in this file as it's a historical archive of newsletters we sent out in the past, so perhaps for consistency it should use the terminology that was current at the time.

sanderginn commented on content/departments/engineering/teams/devinfra/processes.md:
```diff
@@ -214,25 +214,25 @@ Every week we hold an informal, totally optional coffee break to connect sociall
-When the hack hour starts, a meeting link will be posted in #dev-experience.
+When the hack hour starts, a meeting link will be posted in #devinfra.
 
 To learn more about contributing to `sg`, check out the [contribution guide](https://docs.sourcegraph.com/dev/background-information/sg#contributing-to-sg)!
```
This event no longer takes place, so this can be scrapped from the handbook

sanderginn commented on content/departments/engineering/teams/devinfra/processes.md:
```diff
@@ -214,25 +214,25 @@ Every week we hold an informal, totally optional coffee break to connect sociall
 ### Weekly reminders
 
-At the start of each week, the PM posts a reminder in `#dev-experience-internal` indicating who is on support and who is leading meetings.
+At the start of each week, the PM posts a reminder in `#devinfra-internal` indicating who is on support and who is leading meetings.
```
Can be scrapped

sanderginn commented on content/departments/engineering/teams/release/index.md:
```diff
@@ -23,7 +23,7 @@ For more detailed information on the Delivery team members, check out our [READM
 
 - #delivery channel, or `@delivery-support` in Slack. Please follow our [support guidelines](#support-request-guidelines) below.
-- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/delivery) team on GitHub.
+- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/release) team on GitHub.
```
A couple items that need to be renamed `delivery` -> `release` here

sanderginn commented on content/departments/engineering/teams/release/onboarding.md:
```diff
@@ -30,7 +30,7 @@ Welcome to the Delivery team! This document will guide you through delivery-spec
   - #dev-chat
   - #dev-accounce
-- Make sure you are added to the [Delivery](https://github.com/orgs/sourcegraph/teams/delivery) team in Sourcegraph GitHub org. Reach out to [#delivery-internal] if you're not.
+- Make sure you are added to the [Delivery](https://github.com/orgs/sourcegraph/teams/release) team in Sourcegraph GitHub org. Reach out to [#delivery-internal] if you're not.
```
```suggestion
- Make sure you are added to the [Release](https://github.com/orgs/sourcegraph/teams/release) team in Sourcegraph GitHub org. Reach out to [#team-releases] if you're not.
```

Note: the Slack channel is called `#team-releases`, but I see references to `Release` here. I think the plural form is preferred?

jdpleiness replied:
I've seen `Release` everywhere but the Slack channel naming (Miro chart, RFCs, etc...). Either sounds right to me though 🤷

sanderginn commented on content/departments/sales/onboarding/index.md:
```diff
@@ -42,7 +42,7 @@ Read about Sales onboarding Mentor program:
 - [Sales](index.md)
 - [Data and Looker onboarding](data_onboarding.md)
-- [Delivery team](../../engineering/teams/delivery/index.md)
+- [Delivery team](../../engineering/teams/release/index.md)
```
```suggestion
- [Release team](../../engineering/teams/release/index.md)
```

sanderginn commented on content/departments/technical-success/support/process/engaging-other-teams.md:
```diff
@@ -55,7 +55,7 @@ A couple of notes before to help you get started in figuring out which team will
 **Keywords**: `deploy`, `Docker`, `container`, `image`, `Kubernetes`/`k8s`, `cluster`, `AWS`, `Google Cloud`/`GCP`, `self-hosted`
 
-Any questions about self-hosted deployment should be routed to the [Delivery team](../../../engineering/teams/delivery/index.md).
+Any questions about self-hosted deployment should be routed to the [Delivery team](../../../engineering/teams/release/index.md).
```
```suggestion
Any questions about self-hosted deployment should be routed to the [Release team](../../../engineering/teams/release/index.md).
```

sanderginn commented on data/glossary.yml:
```diff
@@ -169,7 +169,7 @@ role:
+  - term: DevInfra
     definition: Developer experience, improving lives of devs working on sourcegraph
   - term: dx
     definition: Developer experience, improving lives of devs working on sourcegraph
```
Couple renames here

sanderginn commented on data/team.yml:
```diff
@@ -1793,8 +1793,8 @@ jacob_pleiness:
-  role: Software Engineer
-  reports_to: head_ship
+  role: Software Engineer, Tech Lead
+  reports_to: ship_release_lead
```
Reporting to himself? 😄

sanderginn reviewed (approved):
I started adding suggestions for renaming channels and handles but there are a lot of them that need to be revisited here, so I stopped calling them out

rafax reviewed (approved):
Added some changes to the Cloud Ops page

jdpleiness commented on content/departments/engineering/teams/release/index.md:
```diff
@@ -1,12 +1,12 @@
-{{generator:product_team.delivery}}
+{{generator:product_team.ship_release}}
 
 For more detailed information on the Delivery team members, check out our [README page](./team/index.md) which expands on our profile.
```
```suggestion
For more detailed information on the Release team members, check out our [README page](./team/index.md) which expands on our profile.
```

jdpleiness commented on content/departments/engineering/teams/release/index.md:
```diff
@@ -23,7 +23,7 @@ For more detailed information on the Delivery team members, check out our [READM
-- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/delivery) team on GitHub.
+- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/release) team on GitHub.
 
 ## Support
```
Does a support rotation even makes sense for the Release team in the new Org structure? Probably outside the scope of this PR but this will either have to be reworked completely or deleted based on that discussion.

jdpleiness commented on content/departments/technical-success/support/process/engaging-other-teams.md:
```diff
@@ -65,7 +65,7 @@ Any questions about managed instance deployment should be routed to the [Cloud t
 Questions about specific alerts and graph panels should be routed to the team that is responsible for the alert or panel, as indicated by relevant entry in [Alert solutions](https://docs.sourcegraph.com/admin/observability/alert_solutions) or the [Dashboards reference](https://docs.sourcegraph.com/admin/observability/dashboards) respectively.
 
-Any other questions about monitoring and performance for self-hosted customers should be routed to the [Delivery team](../../../engineering/teams/delivery/index.md).
+Any other questions about monitoring and performance for self-hosted customers should be routed to the [Delivery team](../../../engineering/teams/release/index.md).
```
```suggestion
Any other questions about monitoring and performance for self-hosted customers should be routed to the [Release team](../../../engineering/teams/release/index.md).
```

jdpleiness reviewed (approved):
Added some fixes to the Delivery/Release team pages.

There's a lot of outdated information regarding Release/Delivery/Delivery Tiger Team that has accumulated over the years and through several reorgs. At some point Release team will need to revisit some of the areas (on-call rotation, onboarding, ownership, etc...)

willdollman committed:
Add placeholder accepted CVEs for 5.1.2 release (#7278)

TODO:
- [x] trivy-diff scan against release candidate
- [x] snyk scan on pre-release-candidate (`:792a406cb5b1_2023-06-30`) -
0 vulns

sqs committed:
Update faq.md

MaedahBatool committed:
Add missing info about Go installation (#7280)

The installation of Go is a missing pre-requisite step in our docs
contributing guide. This PR adds this missing info.

unknwon committed:
Apply suggestions from code review

Co-authored-by: Rafał Gajdulewicz <rafax@users.noreply.github.com>

MaedahBatool committed:
Improve enterprise-vs-oss.md (#7281)

Improve the doc to reflect the difference between Sourcegraph Enterprise
and Cody which is in fact open-source.

unknwon commented on content/departments/cloud/index.md:
```diff
@@ -66,30 +50,30 @@ Build a **fully managed platform** for using Sourcegraph that can **(by EOFY23)
+- You may tag the `@cloud-support` handle if you are looking for immediate attention, and it will notify our [on-call engineers](#on-call). Please avoid tagging/DM a specific teammate or the `@cloud-team` handle.
 
-## Managed Instance
+## Cloud Instances
```
This is an intentional terminology change? There are countless places we use "managed instances" and in reality most of the time people tend to type its abbreviation "MI", which is well-perceived. Where "CI" means... totally something different. @nelsonjr @rafax

rafax replied:
My impression was that we use "Cloud Instance" more often than "Managed Instance" now (vs 1-year ago when this was written), but I don't feel strongly about this change - if you think we should stay with MI, happy to keep that.

unknwon replied:
A quick unscientific stats across our repos 😂 

<img width="596" alt="CleanShot 2023-07-05 at 09 30 25@2x" src="https://github.com/sourcegraph/handbook/assets/2946214/36de3706-d087-46e4-b026-347f1dd556ad">


<img width="609" alt="CleanShot 2023-07-05 at 09 31 01@2x" src="https://github.com/sourcegraph/handbook/assets/2946214/372f2648-0ef7-4df7-927d-c8d64e5eb4bb">

But I'm OK with any term, original intention was to make sure this ("Cloud Instances") is term  we are settling with, so whenever I see the another term, I know I should change/update them for consistency.

unknwon commented on content/departments/cloud/index.md:
```diff
@@ -124,7 +108,7 @@ Customer Engineers (CE) or Sales may request to:
    - For standard managed instance requests (i.e., new instance, no scale concerns, no additional security requirements), CE submits a request to the Cloud team using the corresponding [issue template](#managed-instance-requests) in the [sourcegraph/customer](https://github.com/sourcegraph/customer) repo.
    - For non-standard managed instance requests (i.e., any migrations, special scale or security requirements, or anything considered unusual), CE submits the opportunity to Tech Review before making a request to the Cloud team.
-4. Message the team in [`#cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP).
+4. Message the team in [`#ask-cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP).
```
```suggestion
4. Message the team in [`#discuss-cloud-ops`](https://sourcegraph.slack.com/archives/C03JR7S7KRP).
```

MaedahBatool committed:
Fix Typo in enterprise-vs-oss.md (#7282)

Typo Code → Cody

marijapetrovic214 committed:
update pto process (#7284)

update pto process

kemperhamilton committed:
updating intern perks (#7285)

kemperhamilton committed:
Updating intern info (#7286)

coury-clark committed:
Update out of date information on release guild page (#7287)

michaellzc committed:
update post creation msg (#7289)

marijapetrovic214 committed:
Fix link (#7290)

Fix link

marijapetrovic214 committed:
fix links (#7291)

fix link

marijapetrovic214 committed:
Fix link (#7292)

Fix link

marijapetrovic214 committed:
fix links within page (#7293)

fix links within page

marijapetrovic214 committed:
fix link attempt (#7295)

fix link attempt

jhchabran commented:
Packed a bunch of updates in https://github.com/sourcegraph/handbook/pull/7296

mrnugget committed:
Update org chart to reflect new Source team (#7297)

This is a pretty empty, skeleton change: no big content changes, only
changing pointers and who reports to whom, and so on.

I'll follow-up with content changes in separate PRs to make them easier
to review.

jhchabran committed:
Various dev-infra fixes/updates  (#7296)

#7277 updates (suggestions are nice but so annoying for the PR author 😆)

---------

Co-authored-by: William Bezuidenhout <william.bezuidenhout@sourcegraph.com>

jkoconne committed:
Remove LCA posting.md (#7298)

Removed old LCA postings & updated historical LCA posting link

jkoconne committed:
Updated Links for QLE COBRA and Internal FAQ.md (#7299)

Updated Rippling links for QLE, broken cobra link, and internal links

filiphaftek committed:
Fix CloudSQL DB migration steps (#7294)

burmudar committed:
graphql: add walkthrough video (#7300)

Feel free to suggest a better place for this!

jkoconne committed:
Update page title.md (#7219)

From Required workplace postings to Workplace postings

AJKemps committed:
adds Alex Kemper to team.yml (#7309)

doragrgic committed:
Add new accepted CVE to release 5-1-2.md (#7317)

Added new accepted caddy CVE  into the list for 5.1.2 release

---------

Co-authored-by: doragrgic <doragrgic@users.noreply.github.com>

doragrgic committed:
Correction to image name (#7318)

Correct affected image name and correct detail description.

marijapetrovic214 committed:
delete content (#7319)

delete content

jkoconne committed:
Sequoia link updated.md (#7320)

auppaluri1 committed:
Update salesforce taxonomy updated.md (#7321)

Updating taxonomy to reference Code Search vs. CIP and list Cody as a
Product

---------

Co-authored-by: auppaluri1 <auppaluri1@users.noreply.github.com>

bobheadxi committed:
cody-gateway: better prod logs link (#7322)

This one has better formatting with scope, body, error


![image](https://github.com/sourcegraph/handbook/assets/23356519/98cb3601-eb9a-408a-9748-e10278524f11)

doragrgic committed:
Formatting for CVE line 5-1-2.md (#7323)

Formatting correction for accepted CVE line

nelsonjr committed:
Companion change for sg rfc create. (#7288)

https://github.com/sourcegraph/sourcegraph/issues/25440
https://github.com/sourcegraph/sourcegraph/pull/54644

mrnugget committed:
Update source support rotation (#7316)

This is based on our 5.1 retrospective. It contains the rules that Ryan
proposed and I also added something about support rotation in general.
Most of that is from the old repo-management pages, but adjusted.

---------

Co-authored-by: mrnugget <mrnugget@users.noreply.github.com>

unknwon committed:
cloud: minor cody enable docs improvement (#7324)

Follow up of
https://github.com/sourcegraph/handbook/pull/7289/files#r1254719421

auppaluri1 committed:
Update salesforce typos.md (#7325)

Change slight typos

---------

Co-authored-by: auppaluri1 <auppaluri1@users.noreply.github.com>

jkoconne committed:
Added new travel insurance information.md (#7326)

Co-authored-by: jkoconne <jkoconne@users.noreply.github.com>

michaellzc committed:
update post-creation process (#7327)

TrevorHoughton committed:
Added instructions on cost center and division (#7329)

kemperhamilton committed:
Removing departing teammate (#7331)

kemperhamilton committed:
updating removed teammate (#7332)

Commit:
Add Tech Nurture V1 (#7333)

Co-authored-by: billCaplan <billCaplan@users.noreply.github.com>

auppaluri1 committed:
Update salesforce typos.md (#7334)

Fix one typo

---------

Co-authored-by: auppaluri1 <auppaluri1@users.noreply.github.com>

auppaluri1 committed:
Update salesforce typos.md (#7334)

Fix one typo

---------

Co-authored-by: auppaluri1 <auppaluri1@users.noreply.github.com>

auppaluri1 committed:
Update salesforce typos.md (#7334)

Fix one typo

---------

Co-authored-by: auppaluri1 <auppaluri1@users.noreply.github.com>

mucles committed:
init grpc doc (#7264)

Co-authored-by: ggilmore <geoffrey@sourcegraph.com>
Co-authored-by: ggilmore <ggilmore@users.noreply.github.com>

stephmhay committed:
Specify legacy nature of lead scoring.md (#7336)

stephmhay committed:
Added MOPS process detail for SH transition.md (#7338)

jdorfman committed:
Add instructions for external Cody contributors (#7339)

This is for a `CONTRIBUTING.md` file for the Cody repo.

---------

Co-authored-by: jdorfman <jdorfman@users.noreply.github.com>

nelsonjr committed:
Update content/departments/cloud/index.md

Co-authored-by: Michael Lin <mlzc@hey.com>

nelsonjr committed:
Update content/departments/cloud/technical-docs/v2.0/index.md

Co-authored-by: Michael Lin <mlzc@hey.com>

nelsonjr committed:
Update content/departments/engineering/dev/process/deployments/instances.md

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

nelsonjr committed:
Update content/departments/engineering/teams/release/onboarding.md

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

nelsonjr committed:
Update content/departments/engineering/dev/process/engineering_ownership.md

Co-authored-by: Jacob Pleiness <jdpleiness@users.noreply.github.com>

TrevorHoughton committed:
Removed "growth" from PM overview of past projects (#7340)

To avoid confusion, I removed "growth" from the title of the Product
Manager overview of past projects prompt so that we can also use it for
the Code AI platform role.

amenne committed:
adding cody exception process to handbook (#7341)

jkoconne committed:
Creating ADA page.md (#7342)

Adding ADA page

kemperhamilton committed:
fixing typo (#7343)

michaellzc committed:
update cloud nat for cloud (#7344)

michaellzc committed:
update cloud docs (#7345)

kemperhamilton committed:
Updating Deel statutory minimums FAQs (#7346)

Co-authored-by: kemperhamilton <kemperhamilton@users.noreply.github.com>

TrevorHoughton committed:
Added AI coding exercise (#7348)

Added AI coding exercise for Head of ML Infra role

kemperhamilton committed:
removing outdated google calendar link (#7347)

jkoconne committed:
Revert "Creating ADA page.md" (#7349)

Reverts sourcegraph/handbook#7342

andreeleuterio committed:
Add 5.1.3 vulns (#7350)

Add 5.1.3 vulns

---------

Co-authored-by: andreeleuterio <andreeleuterio@users.noreply.github.com>

nelsonjr committed:
squash! Prettier

nelsonjr committed:
squash! Prettier

unknwon committed:
Update content/departments/cloud/index.md

unknwon committed:
Merge branch 'main' into nelsona/org-ship-1

unknwon committed:
Update content/departments/technical-success/support/process/engaging-other-teams.md

Co-authored-by: Jacob Pleiness <jdpleiness@users.noreply.github.com>

unknwon commented on content/departments/cloud/index.md:
```diff
@@ -66,30 +54,30 @@ Build a **fully managed platform** for using Sourcegraph that can **(by EOFY23)
 
 - For emergencies and incidents, alert the team using Slack command `/genie alert [message] for cloud` and optionally tag the `@cloud-support` handle.
-- For internal Sourcegraph teammates, join us in #cloud slack channel to ask questions or request help from our team.
+- For internal Sourcegraph teammates, join us in #ask-cloud-ops slack channel to ask questions or request help from our team.
```
```suggestion
- For internal Sourcegraph teammates, join us in #discuss-cloud-ops slack channel to ask questions or request help from our team.
```

unknwon commented on content/departments/cloud/index.md:
```diff
@@ -66,30 +54,30 @@ Build a **fully managed platform** for using Sourcegraph that can **(by EOFY23)
 See below for the SLAs and Technical implementation details (including Security) related to managed instances.
 
-Please message [`#cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP) for any answers or information missing from this page.
+Please message [`#ask-cloud`](https://sourcegraph.slack.com/archives/C03JR7S7KRP) for any answers or information missing from this page.
```
```suggestion
Please message [`#discuss-cloud-ops`](https://sourcegraph.slack.com/archives/C03JR7S7KRP) for any answers or information missing from this page.
```

unknwon committed:
Apply suggestions from code review

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon commented on content/departments/cloud/index.md:
```diff
@@ -228,13 +216,13 @@ GCP Logging is used to store / query logs written by Sourcegraph workloads, and
 Cloud instances provisioned for customers run [released](../engineering/dev/process/releases/index.md#releases) Sourcegraph versions and are currently updated at least once a month (for minor releases), unless we need to deploy a patch release.
 
-Sourcegraph-owned instances are continuously deployed (with versions that weren't officially released), [DevX team](../engineering/teams/dev-experience/index.md) owns continuous deployment to those environments.
+Sourcegraph-owned instances are continuously deployed (with versions that weren't officially released), [Developer Infrastructure team](../engineering/teams/devinfra/index.md) owns continuous deployment to those environments.
```
```suggestion
Sourcegraph-owned instances can opt-in to receive continuous upgrades with versions that weren't officially released, e.g., the latest commit on `main` branch, release branches. We own continuous deployment to those environments.
```

unknwon committed:
Apply suggestions from code review

unknwon commented on content/departments/cloud/technical-docs/v2.0/index.md:
```diff
@@ -72,7 +72,7 @@ All customer instances are considered part of the `prod` environment and all cha
 ##### [s2 instance](https://sourcegraph.sourcegraph.com/)
 
-This is the internal Cloud dogfood instance for the entire company. #dev-experience is responsible for rolling out nightly builds on this instance and #cloud is responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
+This is the internal Cloud dogfood instance for the entire company. #ask-cloud-ops is responsible for rolling out nightly builds on this instance. Additionally, they are responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
```
```suggestion
This is the internal Cloud dogfood instance for the entire company. #discuss-cloud-ops is responsible for rolling out nightly builds on this instance. Additionally, they are responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
```

unknwon committed:
Update content/departments/cloud/technical-docs/v2.0/index.md

unknwon committed:
Update index.md

unknwon committed:
Apply suggestions from code review

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon commented on content/departments/engineering/dev/process/incidents/playbooks/ci.md:
```diff
@@ -1,10 +1,10 @@
 # Continuous integration playbook
 
-- **Maintainers**: [DevX Team](../../../../teams/dev-experience/index.md).
+- **Maintainers**: [DevInfra Team](../../../../teams/dev-experience/index.md).
```
```suggestion
- **Maintainers**: [DevInfra Team](../../../../teams/devinfra/index.md).
```

unknwon committed:
Update content/departments/engineering/dev/process/incidents/playbooks/ci.md

unknwon committed:
Apply suggestions from code review

unknwon committed:
Update content/departments/engineering/dev/process/incidents/playbooks/ci.md

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon committed:
Apply suggestions from code review

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon committed:
Update content/departments/engineering/dev/process/incidents/playbooks/ci.md

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon committed:
Update pullrequest-compliance.md

unknwon committed:
Update scaletesting.md

unknwon committed:
Update index.md

unknwon committed:
Update newsletter.md

unknwon committed:
Update processes.md

unknwon commented on content/departments/engineering/teams/devops/update_sg_website_config.md:
```diff
@@ -23,7 +23,7 @@ To update the non-sensitive configuration, follow these steps:
 Our site configuration contains many secrets like OAuth credentials. It is [stored in GSM](https://console.cloud.google.com/security/secret-manager/secret/SITE_JSON/versions?project=sourcegraph-dev) in the `sourcegraph-dev` project. The secrets are synced to the cluster using [Terraform](https://sourcegraph.sourcegraph.com/github.com/sourcegraph/infrastructure/-/blob/cloud/gsm-secrets.tf), and is ...
 
-> NOTE: syncing new secret versions requires access to Terraform Cloud. Reach out to #ask-security to get access, or request #ask-dev-experience or #ask-security to run the workspace for you.
+> NOTE: syncing new secret versions requires access to Terraform Cloud. Reach out to #ask-security to get access, or request #ask-devinfra or #ask-security to run the workspace for you.
```
```suggestion
> NOTE: syncing new secret versions requires access to Terraform Cloud. Reach out to #ask-security to get access, or request #discuss-dev-infra or #ask-security to run the workspace for you.
```

unknwon committed:
Update content/departments/engineering/teams/devops/update_sg_website_config.md

unknwon committed:
Update content/departments/engineering/teams/release/index.md

Co-authored-by: Jacob Pleiness <jdpleiness@users.noreply.github.com>

unknwon commented on content/departments/engineering/teams/release/index.md:
```diff
@@ -23,7 +27,7 @@ For more detailed information on the Delivery team members, check out our [READM
 
 - #delivery channel, or `@delivery-support` in Slack. Please follow our [support guidelines](#support-request-guidelines) below.
-- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/delivery) team on GitHub.
+- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/release) team on GitHub.
```
```suggestion
- [team/release](https://github.com/sourcegraph/sourcegraph/labels/team%2Frelease) label and [@sourcegraph/release](https://github.com/orgs/sourcegraph/teams/release) team on GitHub.
```

unknwon committed:
Update content/departments/engineering/teams/release/index.md

unknwon committed:
Update content/departments/sales/onboarding/index.md

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon commented on content/departments/technical-success/support/process/enablement/index.md:
```diff
@@ -40,7 +40,7 @@ This table correlates to our [technical competency matrix](https://docs.google.c
 - [What does upgrading Sourcegraph look like](https://drive.google.com/file/d/1oIrKJPo9hvc2UMfpb-a57HV9fCYEcS9n/view)
 - [What does upgrading a Kubernetes instance with overlay look like](https://drive.google.com/file/d/1aWsXejG2qeFKGrY5BGzpSabwGuU1gLM3/view?usp=sharing)
-- [How to deploy Sourcegraph to GCP with Kubernetes](https://drive.google.com/file/d/10uIp-rcN3nRa0FguScHU3NRrcXxgy6C7/view?usp=sharing) _note that #distribution is now defunct and if you have further questions on how to get up a cluster you should consult #delivery , #dev-ops and/or #dev-experience_
+- [How to deploy Sourcegraph to GCP with Kubernetes](https://drive.google.com/file/d/10uIp-rcN3nRa0FguScHU3NRrcXxgy6C7/view?usp=sharing) _note that #distribution is now defunct and if you have further questions on how to get up a cluster you should consult #delivery , #dev-ops and/or #devinfra_
```
```suggestion
- [How to deploy Sourcegraph to GCP with Kubernetes](https://drive.google.com/file/d/10uIp-rcN3nRa0FguScHU3NRrcXxgy6C7/view?usp=sharing) _note that #distribution is now defunct and if you have further questions on how to get up a cluster you should consult #discuss-dev-ops and/or #discuss-dev-infra.
```

unknwon committed:
Update content/departments/technical-success/support/process/enablement/index.md

unknwon committed:
Update content/departments/technical-success/support/process/engaging-other-teams.md

Co-authored-by: Sander Ginn <sanderginn@users.noreply.github.com>

unknwon committed:
squash! Prettier

unknwon committed:
Update glossary.yml

unknwon reviewed (approved):
Did a pass to the best of my knowledge, I'm sure I missed few, every team should be responsible for keeping their teams' page up-to-date going forward.

unknwon commented on content/departments/engineering/dev/process/deployments/index.md:
```diff
@@ -128,7 +128,7 @@ In order to mimic the same workflow that we tell our customers to follow:
 ## DotCom
 
-As [DotCom](instances.md#dotcom) has matured into our public, free-to-use product its needs have diverged from the base `deploy-sourcegraph` repo it initially was a fork of. It no longer maintains `deploy-sourcegraph` as a remote and infrastructure and deployment changes are now manually merged in as-needed by the [Dev Experience](../../../teams/dev-experience/index.md) team.
+As [DotCom](instances.md#dotcom) has matured into our public, free-to-use product its needs have diverged from the base `deploy-sourcegraph` repo it initially was a fork of. It no longer maintains `deploy-sourcegraph` as a remote and infrastructure and deployment changes are now manually merged in as-needed by the [Developer Infrastructure](../../../teams/dev-experience/index.md) team.
```
```suggestion
As [DotCom](instances.md#dotcom) has matured into our public, free-to-use product its needs have diverged from the base `deploy-sourcegraph` repo it initially was a fork of. It no longer maintains `deploy-sourcegraph` as a remote and infrastructure and deployment changes are now manually merged in as-needed by the [Developer Infrastructure](../../../teams/devinfra/index.md) team.
```

unknwon commented on content/departments/engineering/dev/tools/infrastructure/ci/index.md:
```diff
@@ -1,7 +1,7 @@
 
 This page consolidates resources regarding our CI _infrastucture_, namely our [Buildkite agents fleet](#buildkite-agents).
-This infrastructure is maintained by the [DevX team](../../../../teams/dev-experience/index.md).
+This infrastructure is maintained by the [DevInfra team](../../../../teams/dev-experience/index.md).
```
```suggestion
This infrastructure is maintained by the [DevInfra team](../../../../teams/devinfra/index.md).
```

unknwon commented on content/departments/engineering/teams/devinfra/index.md:
```diff
@@ -1,24 +1,28 @@
 ## Strategy
 
-Find out more about the Dev Experience team's mission, vision, and strategic plans in our [Strategy page](../../../../strategy-goals/strategy/dev-experience/index.md).
+Find out more about the Developer Infrastructure team's mission, vision, and strategic plans in our [Strategy page](../../../../strategy-goals/strategy/dev-experience/index.md).
```
```suggestion
Find out more about the Developer Infrastructure team's mission, vision, and strategic plans in our [Strategy page](../../../../strategy-goals/strategy/devinfra/index.md).
```

unknwon commented on content/departments/engineering/teams/devinfra/newsletter.md:
```diff
@@ -1,8 +1,8 @@
 # 🗞 Developer experience newsletter
 
-Welcome to the developer experience newsletter! This is a newsletter prepared by the [DevX team](./index.md) to highlight contributions and updates to Sourcegraph's developer experience, which is [an area the DevX team focuses on but is owned by everyone](../../../../strategy-goals/strategy/dev-experience/index.md#guiding-principles).
+Welcome to the developer experience newsletter! This is a newsletter prepared by the [DevInfra team](./index.md) to highlight contributions and updates to Sourcegraph's developer experience, which is [an area the DevInfra team focuses on but is owned by everyone](../../../../strategy-goals/strategy/dev-experience/index.md#guiding-principles).
```
```suggestion
Welcome to the developer experience newsletter! This is a newsletter prepared by the [DevInfra team](./index.md) to highlight contributions and updates to Sourcegraph's developer experience, which is [an area the DevInfra team focuses on but is owned by everyone](../../../../strategy-goals/strategy/devinfra/index.md#guiding-principles).
```

unknwon committed:
Update content/departments/engineering/dev/process/deployments/index.md

unknwon committed:
Update content/departments/engineering/teams/devinfra/index.md

unknwon committed:
Update content/departments/engineering/dev/tools/infrastructure/ci/index.md

unknwon committed:
Update content/departments/engineering/teams/devinfra/newsletter.md
